Makefile

```diff
@@ -22,3 +32,6 @@ requirements:
 test:
 	# Confirm compiled assets have not changed, indicating SASS matches CSS.
 	git diff --exit-code edx_credentials_themes/
+
+validate_translations:
```

Comment:
It would be ideal if all the repos' targets behaved similarly. See https://github.com/edx/course-discovery/blob/3900dda2ae0fd6cacf9d8b012319e76856a93617/Makefile#L97.

Reply:
I would prefer to update the other repos to be more like this one. Names like `fake_translations` do not really communicate what is going on, and it is not immediately clear why we would call a command with that name from within the validate command. Furthermore, the other repos rely on `compilemessages` to detect problems with translation files, which does not detect things like HTML tag mismatches.